packages/core/src/System.js

```diff
@@ -18,26 +18,13 @@ export default class System
          * @member {PIXI.Renderer}
          */
         this.renderer = renderer;
```

Comment:
hello! Not sure i fully understand why we want to do this? There are many systems that hook into the runner?